Fleece/Support/ParseDate.cc

```diff
@@ -431,10 +431,11 @@ namespace fleece {
     DateTime FromMillis(int64_t timestamp) {
         // Split out the milliseconds from the timestamp:
         time_t secs{timestamp / 1000};
-        int    millis = (int)timestamp % 1000;
+        int    millis = (int)(timestamp % 1000);
```

Comment:
Do you need to narrow_cast it?

Reply:
If it is % 1000 it is guaranteed not to overflow or underflow.

Reply:
max of int64_t is roughly 2^63, and max of int (assuming 32 bit is roughly 2^31.